src/SerialInputHandler.cpp

```diff
@@ -5,6 +5,7 @@
 
 #include <ArduinoJson.h>
 #include <Esp.h>
+#include "Logging.h"
```

Comment:
minor issue, but not big enough to require a change.

Try to order these in the order of:
Header file
Our headers
Library headers
Esp headers
C++ headers

Reply:
This is due to in some cases having the improper include order can mess up compilation in really weird ways that are incredibly hard to debug 😅

Reply:
where do i put this header here then?

Reply:
Its not that important, but you would put it alongside "CommandHandler.h" and "Config.h"

Reply:
okay, but its not one of our headers? so you initial proposal doesnt apply here?

Reply:
Logging.h *is* one of our headers